docs/theme_architecture/blocks/schema/variables/variable_types/experience_date.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Experience Date
+parent: Variable types
+has_children: false
+---
+
+# Experience Date
+
+Renders two select inputs from which a Creator can select an experience and then a date that has availability for the chosen experience.
```

Comment:
"A date that has availability" - does this mean that a creator will not be able to display an experience that is sold out or has no variants?

Reply:
Maybe I should reword this. The creator will be able to display experiences that are sold out but not if they have no variants. The dropdown will allow selecting any experience, even if it has no variants. However, the `date.slot_count` would be 0 and `slots_by_variant` would be empty.

Is there a situation where a creator will want to display an experience that has no variants? I don't think that makes sense for this variable, as this is used to display the price and availability of variants on particular slots.

I'll reword to the following:

> Renders two select inputs. First, the creator can select an experience. Then they will be prompted to choose a slot date for the chosen experience.